src/test/java/org/broadinstitute/hellbender/engine/VariantWalkerBaseUnitTest.java

```diff
@@ -0,0 +1,44 @@
+package org.broadinstitute.hellbender.engine;
+
+import htsjdk.variant.variantcontext.VariantContext;
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.File;
+
+public class VariantWalkerBaseUnitTest extends CommandLineProgramTest {
```

Comment:
When you move the `getIntervals()` method up into `GATKTool` as suggested above, this test case can move into `GATKToolIntegrationTest`

Reply:
I dont follow here.  @cmnbroad suggested that I use the previous computed intervalsForTraversal (see earlier version); however, who is suggesting moving to GATKTool?

Reply:
I suggested it above in https://github.com/broadinstitute/gatk/pull/4495#discussion_r189694685

Reply:
got it.  I moved the test and extended it to cover some situations that are not possible for VariantWalker, but could occur for GATKTool